src/main.js

```diff
@@ -1 +1,229 @@
-// Write your code here
\ No newline at end of file
+// Write your code here
+
+// task 1
+// 1.What method would you use to add an element to the end of an array?
+// for example 
+const pushArray = [1, 4, 5];
+pushArray.push(10, 10, 10);
+console.log(pushArray);
+
+// task 2 
+// How do you remove the first element from an array?
+const removeArray = [10, 9, 8];
+removeArray.shift();
+console.log(removeArray);
+
+// task 3 
+
+// დამატების ან ამოღების შემთხევაში ვიყენებთ splice
+
+// const arr = [1, 2, 3, 4, 5];
+// arr.splice(2, 2, 6, 7);  // arr ხდება [1, 2, 6, 7, 5]
+// console.log(arr);
+
+//    თუ უნდა ავირჩიოთ ან შევიანხოთ ახალ მასივში ვიყენებთ slice 
+// const arr = [1, 2, 3, 4, 5];
+// let newArr = arr.slice(1, 4);  // newArr იქნება [2, 3, 4]
+// console.log(newArr);
+
+// task 4 
+const arr = [10, 20, 30, 40, 20, 60, 70, 80];
+console.log(arr.length);
+// იქნება 8 
+
+// task 5
+
+
+console.log(arr.includes(20));
+console.log(arr.includes(50));
+
+// task 6 
+const sortArray = [100, 1, 1000, 10, 20, 2000, 200, 2];
+
+sortArray.sort();
+console.log(sortArray);
+
+// task 7 
+let copyArrey = [1, 10, 100];
+let copy = copyArrey.slice();
+console.log(copy);
+
+// task 8 
+const reversArray = [1, 2, 3, 4, 5, 6];
+
+reversArray.reverse();
+console.log(reversArray);
+
+// task 9 
+let firstArray = [10, 20, 30];
+let secendArray = [30, 20, 10];
+let combinedArray = firstArray.concat(secendArray);
+
+console.log(combinedArray);
+
+
+// task 10
+
+const matrix = [
+    [10, 20, 30],  // first line is index: 0;  a0 a1 a2
+    [40, 50, 60],  // secend line is index: 1; b0 b1 b2
+    [70, 80, 90]   // third line is index; 2   c0 c1 c2
+];
+//sadghac xo mainc gamomadga matricebis codna 
+
+let matrixElement = matrix[1][1];
+console.log(matrixElement);
+
+
+
+// Assignment 1: Basic Array Operations
+
+const numberArray = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10,];
+
+let thirdElement = numberArray[2];
+console.log(thirdElement);
+
+numberArray[4] = 15;
+console.log(numberArray);
+
+numberArray.push(100);
+console.log(numberArray);
+
+numberArray.shift();
+console.log(numberArray);
+
+// Assignment 2: Array Methods - push, pop, shift, unshift
+
+const Fruits = ["Apple", "Banana", "Kiwi"];
```

Comment:
ცვლადს სახელები სწორად დაარქვი
```suggestion
const fruits = ["Apple", "Banana", "Kiwi"];
```